client/src/components/entry.jsx

```diff
@@ -101,6 +100,9 @@ class Entry extends React.Component {
         <DlcName className='dlcName' onMouseOver={this.handleToggle} onMouseOut={this.handleToggle}>
           {this.props.data.title}
         </DlcName>
+        <Hover left={this.state.left} top={'-38px'} data={this.state.hover.toString()}>
```

Comment:
Looks like you're changing hover state from boolean to string before passing it to your Hover styled component, but it's unnecessary. The Hover styled component will accept any data type for its props, meaning `data={this.state.hover}` and `display: ${props => props.data ? 'block' : 'none'}` will work fine. (note: changed `visible` to `block` in this example here as mentioned in my other comment)